### DISCUSSION TITLE
Icons needed for RBAC designs

### DISCUSSION THREAD

**Discussion by pguptaDS** [twilio-labs/paste#864] [Request: Icon] [answered]
We want to use the following icons in RBAC Designs to indicate "Built-in" roles and "Custom" roles. 

*Built-in* roles - 
(from the controls section)
![image](https://user-images.githubusercontent.com/64559618/96300198-00f75c00-0faa-11eb-8e35-dd21ad712928.png)

*Custom* roles -
(from the edit  section)
![image](https://user-images.githubusercontent.com/64559618/96300366-4451ca80-0faa-11eb-82ff-ae89e277e9ae.png)

**Comment by serifluous** [comment]
Do you need these to be specific to RBAC? Or could they be used for any "built-in" or "preset" use case and any "custom" use case in our products?

**Comment by pguptaDS** [comment]
I don't need them to be specific to RBAC. If other teams want to use these icons for similar use cases, we'd be ok with that.

**Comment by serifluous** [comment, marked as answer]
Chatted with @pguptaDS on Slack, and we'll add these in our next sprint as "BuiltInIcon" and "CustomIcon"

**Comment by serifluous** [comment]
Hey @pguptaDS  ! BuiltInIcon and CustomIcon are available in Paste now: https://paste.twilio.design/icons/

The icons aren't available yet in Sketch, though. We're going to merge them in during our next release (Nov 10).

**Reply by pguptaDS** [reply]
Sounds good! Thanks, Sarah!
